uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -141,22 +141,33 @@ <h4 class="modal-title">New Document</h4>
                 </div>
               </div>
             </div>
+
+            <!-- Display only for manager role and hidden removed from DOM for other uasers -->
+            <div class="col-xs-6" ng-if="isRole('manager')">
```

Comment:
Well, thanks for catching! Since we wont be needing the Assignments tab, for both users (staff and manager), it's more appropriate not to keep it in the DOM. So, this
```html
<div class="tab-pane" id="assignmentsTab" ng-hide="isRole('staff') || isRole('manager')">
```
should be 
```html
<div class="tab-pane" id="assignmentsTab" ng-if="!isRole('staff') && !isRole('manager')">
```